server/server.go

```diff
@@ -344,6 +344,18 @@ func New(ctx context.Context, config *Config) (*Server, error) {
 		return nil, err
 	}
 
+	if config.ClearStorageOnUpgrade {
+		upgrade, err := version.ShouldCrioUpgrade(CrioVersionPath)
+		if err != nil {
+			logrus.Errorf(err.Error())
+		}
+		if upgrade {
+			logrus.Debugf("New version detected, upgrading")
+			if err = s.Store().Wipe(); err != nil {
```

Comment:
We need to ensure is that we don't race against any podman containers. (We can test that out.)

Reply:
Updated to only wipe crio containers/pods :)